src/transformers/onnx/config.py

```diff
@@ -235,6 +235,22 @@ def _generate_dummy_images(
             images.append(Image.fromarray(data.astype("uint8")).convert("RGB"))
         return images
 
+    def _generate_dummy_bbox(self, batch_size: int = 2, image_height: int = 40, image_width: int = 40) -> List[int]:
```

Comment:
I have added a way to generate `dummy_bbox` if required by the preprocessor. The bounding box are scaled on the image size by default.